syntax = "proto3";
package userData;

service userDataRequest
{
    rpc userRequest(Cpf) returns (Data) {}; 
}

message Cpf
{
    string cpf = 1;
}

message Data
{
    string name = 1;
    string cpf = 2;
    string email = 3;
    string address = 4;
}

